includes/collection/class-following.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Following collection file.
+ *
+ * @package Activitypub
+ */
+
+namespace Activitypub\Collection;
+
+use Activitypub\Http;
+
+/**
+ * ActivityPub Following Collection.
+ */
+class Following {
+	/**
+	 * Meta key for the followers user ID.
+	 *
+	 * @var string
+	 */
+	const FOLLOWING_META_KEY = '_activitypub_followed_by';
+
+	/**
+	 * Meta key for pending followers user ID.
+	 *
+	 * @var string
+	 */
+	const PENDING_META_KEY = '_activitypub_followed_by_pending';
+
+	/**
+	 * Follow a user.
+	 *
+	 * @param \WP_Post|int $post    The ID of the remote Actor.
+	 * @param int          $user_id The ID of the WordPress User.
+	 *
+	 * @return \WP_Post|\WP_Error The ID of the Actor or a WP_Error.
+	 */
+	public static function follow( $post, $user_id ) {
+		$post = \get_post( $post );
+
+		if ( ! $post ) {
+			return new \WP_Error( 'activitypub_remote_actor_not_found', 'Remote actor not found' );
```

Comment:
With `Actors::lookup_remote_by_uri()` available, I think it would be great to accept a URI and call that here instead of bailing, following the idea of APIs being permissible with inputs.

Reply:
As I said, it is exactly the same mechanism as with the Outbox. The function is "only" meant to add the follow connection between an already stored `Actor` and a WordPress User. Maybe we would have to rename it then?